pkg/resourcecreator/resource/ast.go

```diff
@@ -17,24 +18,25 @@ type Ast struct {
 
 	// For podSpec
 	Annotations    map[string]string
-	Containers     []v1.Container
-	Env            []v1.EnvVar
-	EnvFrom        []v1.EnvFromSource
-	InitContainers []v1.Container
+	Containers     []corev1.Container
+	Env            []corev1.EnvVar
+	EnvFrom        []corev1.EnvFromSource
+	InitContainers []corev1.Container
+	JobSpec        batchv1.JobSpec
```

Comment:
Det føles feil å ha `JobSpec` her.. alternativt, siden denne er her, hvorfor er ikke `DeploymentSpec` her også?

Reply:
Ikke uenig, kan egentlig lage `JobSpec` når vi trenger den. Blir uansett bare lagd en gang, så ingen krise at det er to steder vi kaller på funksjonen som lager den.